astronomer/providers/apache/livy/triggers/livy.py

```diff
@@ -0,0 +1,133 @@
+"""This module contains the Apache Livy Trigger."""
+import asyncio
+from typing import Any, AsyncIterator, Dict, Optional, Tuple, Union
+
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+
+from astronomer.providers.apache.livy.hooks.livy import BatchState, LivyHookAsync
+
+
+class LivyTrigger(BaseTrigger):
+    """
+    Check for the state of a previously submitted job with batch_id
+
+    :param batch_id: Batch job id
+    :param spark_params: Spark parameters
```

Comment:
An example would be good on what sort of params

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/178/commits/18c4719ba9c58ed4d5a2e779ed4c660a258c75ee)